design.md

```diff
@@ -0,0 +1,38 @@
+# Epiverse visual map
+
+## User persona & user story
+
+This work is mostly targeted at the following persona:
+
+- PEI tool implementer
+- PEI tool creator
+- domain expert
+
+with the following user story:
+
+> I want to quickly & visually understand what Epiverse is about, and its niche in the ecosystem of data science tools for Epidemiology
+
+## Design options and user feedback
+
+3 options have been identified to address the user stories:
+
+- cards
+- network
+- map
+i
+These three options, with their pros and cons have been presented to a panel of potential and they recommended moving forward with a hybrid of the network and cards options.
+
+The network is a good way to get a quick high-level overview of the ecosystem, and Epiverse position in the ecosystem. The cards allow us to see the details of each package.
+
+## Requirements
+
+The tool should:
+
+- represent R packages from the CRAN Task View in Epidemiology as nodes of the network
+- highlight Epiverse packages in a different color
+- allow clicking on a node to see the details of the package
```

Comment:
Would it be helpful to scope out minimum details needed here? Will help to clarify the data to be collected.

Reply:
```suggestion
- allow clicking on a node to see the details of the package via a card. The card should display:
  - the package name
  - the package logo (if it exists)
  - the package description
  - a link to the documentation website (if it exists)
  - a link to the GitHub repository (if it exists)
```

How does this sound?

Reply:
Perfect! 😊